### ISSUE TITLE
[Feature Request] Auto expand post in home feed

### ISSUE THREAD

**Issue by dtantono** [pfgithub/threadclient#11]
Hi
Is it possible to expand the post on home feed rather than click show manually?

**Comment by pfgithub** [comment]
There's an experimental 'fullscreen' view that shows everything expanded: https://thread.pfg.pw/#reddit?--tc-view=fullscreen

Autoexpand shouldn't be too hard to implement

**Comment by dtantono** [comment]
yeah it is. But, unlike the normal one. I cannot login on the experimental one
